R/drive_mv.R

```diff
@@ -52,8 +52,13 @@ drive_mv <- function(file, path = NULL, name = NULL, verbose = TRUE) {
   }
 
   if (is_path(path)) {
-    if (is.null(name) && drive_path_exists(append_slash(path))) {
-      path <- append_slash(path)
+    if (is.null(name) && !has_slash(path) && drive_path_exists(append_slash(path))) {
+      stop(
+        "Unclear if `path` specifies parent folder or full path\n",
+        "to the new file, including its name. ",
+        "See ?as_dribble() for details.",
+        call. = FALSE
+      )
     }
```

Comment:
It feels like we should have a helper and reuse it in multiple places. For the check or for the message.